rules/integrations/google_workspace/initial_access_external_user_added_to_google_workspace_group.toml

```diff
@@ -0,0 +1,113 @@
+[metadata]
+creation_date = "2023/02/16"
+integration = ["google_workspace"]
+maturity = "production"
+min_stack_comments = "Breaking changes introduced to Google Workspace integration"
+min_stack_version = "8.4.0"
+updated_date = "2023/02/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects when an external Google Workspace user account is added to an existing group. Adversaries may add external user
+accounts as a means to intercept shared files or emails with that specific group.
+"""
+false_positives = [
+    """
+    Administrators may add external users to groups to share files and communication with them via the intended
+    recipient be the group they are added to. It is unlikely an external user account would be added to an
+    organization's group where administrators should create a new user account.
+    """,
+]
+from = "now-130m"
+index = ["filebeat-*", "logs-google_workspace*"]
+interval = "10m"
```

Comment:
why such a significant lookback over a short interval?

Reply:
Originally, Google Workspace had latency issues with logs depending on API. Reviewing the documentation now, it seems that is no longer the case for some of the APIs, Admin in this instance is now real-time.

As a result, by default the integration pulls data every 2 hours unless specified otherwise. Thus, our rules typically have this from and interval combination.

Reference: https://github.com/elastic/integrations/blob/807a184a08b0c172b06d70016e0e8a1e13123c5c/packages/google_workspace/data_stream/admin/manifest.yml#L13

Reference: https://support.google.com/a/answer/7061566

Reply:
Can we lengthen the interval then? If no new data for ~2 hours, running every 10 mins seems like wasted resources. Thoughts?

Reply:
@brokensound77 - What would you recommend for this? I can take this approach and apply to the other rules as well.